application/dashboard/views.py

```diff
@@ -46,7 +46,7 @@ def published():
 @login_required
 @user_can(VIEW_DASHBOARDS)
 def measures_list():
-    topics = page_service.get_all_topics()
+    topics = page_service.get_topics(include_testing_space=False)
```

Comment:
I'd be tempted to not have a default value for this, to force people to think about whether or not they want the testing space every time they use it.

Reply:
Yep, sounds reasonable. I tried to leave it with a 'safe' default of public only - but probably your idea is better.